spec/attribute.dd

```diff
@@ -77,6 +77,7 @@ $(GRAMMAR
 $(GNAME LinkageAttribute):
     $(D extern) $(D $(LPAREN)) $(GLINK LinkageType) $(D $(RPAREN))
     $(D extern) $(D $(LPAREN)) $(D C++), $(GLINK2 declaration, IdentifierList) $(D $(RPAREN))
+    $(D extern) $(D $(LPAREN)) $(D C++), $(GLINK StringLiteralList) $(D $(RPAREN))
```

Comment:
It also accepts an expression now, since @TurkeyMan 's PR was merged.

Reply:
StringExp list...?

Reply:
No?